plenum/server/node.py

```diff
@@ -2414,8 +2429,10 @@ def doDynamicValidation(self, request: Request, req_pp_time: int):
 
         # specific validation for the request txn type
         operation = request.operation
-        req_handler = self.get_req_handler(txn_type=operation[TXN_TYPE])
-        req_handler.validate(request)
+        req_manager = self._get_manager_for_txn_type(txn_type=operation[TXN_TYPE])
```

Comment:
What about TAA validation? Should we validate it in `write_req_manager` as well?